src/Kernel32/Kernel32.cs

```diff
@@ -1202,6 +1203,69 @@ public static partial class Kernel32
+        /// </param>
+        /// <param name="flProtect">Specifies the page protection of the file mapping object. All mapped views of the object must be compatible with this protection.</param>
+        /// <param name="dwMaximumSizeHigh">The high-order DWORD of the maximum size of the file mapping object.</param>
+        /// <param name="dwMaximumSizeLow">
+        /// The low-order DWORD of the maximum size of the file mapping object.
+        /// If this parameter and dwMaximumSizeHigh are 0 (zero), the maximum size of the file mapping object is equal to the current size of the file that hFile identifies.
+        /// An attempt to map a file with a length of 0 (zero) fails with an error code of ERROR_FILE_INVALID. Applications should test for files with a length of 0 (zero) and reject those files.
+        /// </param>
+        /// <param name="lpName">
+        /// The name of the file mapping object.
+        /// If this parameter matches the name of an existing mapping object, the function requests access to the object with the protection that flProtect specifies.
+        /// If this parameter is NULL, the file mapping object is created without a name.
+        /// If lpName matches the name of an existing event, semaphore, mutex, waitable timer, or job object, the function fails, and the GetLastError function returns ERROR_INVALID_HANDLE. This occurs because these objects share the same namespace.
+        /// The name can have a "Global" or "Local" prefix to explicitly create the object in the global or session namespace. The remainder of the name can contain any character except the backslash character (\). Creating a file mapping object in the global namespace from a session other than session zero requires the SeCreateGlobalPrivilege privilege. For more information, see Kernel Object Namespaces.
+        /// Fast user switching is implemented by using Terminal Services sessions. The first user to log on uses session 0 (zero), the next user to log on uses session 1 (one), and so on. Kernel object names must follow the guidelines that are outlined for Terminal Services so that applications can support multiple users.
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is a handle to the newly created file mapping object.
+        /// If the object exists before the function call, the function returns a handle to the existing object (with its current size, not the specified size), and GetLastErrorreturns ERROR_ALREADY_EXISTS.
+        /// If the function fails, the return value is NULL. To get extended error information, call GetLastError.
+        /// </returns>
+        [DllImport(nameof(Kernel32), SetLastError = true)]
+        public static extern SafeObjectHandle CreateFileMapping(IntPtr hFile, IntPtr lpAttributes, PageProtection flProtect, int dwMaximumSizeHigh, int dwMaximumSizeLow, string lpName);
```

Comment:
`lpAttributes` should be typed as `SECURITY_ATTRIBUTES*`.
Given the already complicated nature of the maximum size pair of parameters, lets keep them unsigned (`uint`) to match the native headers.